sdk/cosmos/azure-cosmos-spark_3_2-12/docs/configuration-reference.md

```diff
@@ -56,6 +56,7 @@
 | `spark.cosmos.proactiveConnectionInitialization`                  | None    | Can be used to define a list (semicolon separated) of `DB/Container` pairs. Conenctions for these containers will be proactively warmed-up when using direct mode. The format of the config would be `DB1/Collection1;DB2/Collection2` etc.                                                                                                                                                                                                                                                                                                                                                                   |
 | `spark.cosmos.proactiveConnectionInitializationDurationInSeconds` | '120'   | The maximum duration for which the client when being initialized would aggresisvely try to warm-up collections. After this time perios the warm-up will happen only slowly (on one background thread).                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `spark.cosmos.metadata.feedRange.refreshIntervalInMinutes`        | None            | The time interval in minutes to refresh the internal partition key range cache.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Should be seconds with min value of 60?

Reply:
updated from `spark.cosmos.metadata.feedRange.refreshIntervalInMinutes` -> `spark.cosmos.metadata.feedRange.refreshIntervalInSeconds`, and increased default value from 1min to 2min